tests/bcl-test/watchOS-monotouch_Microsoft.CSharp_xunit-test.dll.ignore

```diff
@@ -4,3 +4,150 @@ Microsoft.CSharp.RuntimeBinder.Tests.DefaultParameterTests.MarshalAsOptionalsCor
+Microsoft.CSharp.RuntimeBinder.Tests.IntegerBinaryOperationTests.ConstantDivideByZero(type: Divide, x: 3)
+Microsoft.CSharp.RuntimeBinder.Tests.IntegerBinaryOperationTests.ConstantDivideByZero(type: Modulo, x: 3)
+Microsoft.CSharp.RuntimeBinder.Tests.IntegerBinaryOperationTests.ConstantDivideByZero(type: Modulo, x: 3)
+Microsoft.CSharp.RuntimeBinder.Tests.IntegerBinaryOperationTests.ConstantDivideByZero(type: Divide, x: 3)
+Microsoft.CSharp.RuntimeBinder.Tests.IntegerBinaryOperationTests.ConstantDivideByZero(type: Divide, x: 0)
+Microsoft.CSharp.RuntimeBinder.Tests.IntegerBinaryOperationTests.ConstantDivideByZero(type: Divide, x: -2147483648)
+Microsoft.CSharp.RuntimeBinder.Tests.IntegerBinaryOperationTests.ConstantDivideByZero(type: Divide, x: 3)
+Microsoft.CSharp.RuntimeBinder.Tests.IntegerBinaryOperationTests.ConstantDivideByZero(type: Modulo, x: 2147483647)
+Microsoft.CSharp.RuntimeBinder.Tests.IntegerBinaryOperationTests.ConstantDivideByZero(type: Divide, x: 4294967295)
+Microsoft.CSharp.RuntimeBinder.Tests.IntegerBinaryOperationTests.ConstantDivideByZero(type: Modulo, x: 3)
+Microsoft.CSharp.RuntimeBinder.Tests.IntegerBinaryOperationTests.ConstantDivideByZero(type: Modulo, x: 1)
+Microsoft.CSharp.RuntimeBinder.Tests.IntegerBinaryOperationTests.ConstantDivideByZero(type: Modulo, x: 0)
+Microsoft.CSharp.RuntimeBinder.Tests.IntegerBinaryOperationTests.ConstantDivideByZero(type: Modulo, x: 0)
+Microsoft.CSharp.RuntimeBinder.Tests.IntegerBinaryOperationTests.ConstantDivideByZero(type: Divide, x: 1)
+Microsoft.CSharp.RuntimeBinder.Tests.IntegerBinaryOperationTests.ConstantDivideByZero(type: Divide, x: 3)
+Microsoft.CSharp.RuntimeBinder.Tests.IntegerBinaryOperationTests.ConstantDivideByZero(type: Modulo, x: 0)
+Microsoft.CSharp.RuntimeBinder.Tests.IntegerBinaryOperationTests.ConstantDivideByZero(type: Divide, x: -1)
+Microsoft.CSharp.RuntimeBinder.Tests.IntegerBinaryOperationTests.ConstantDivideByZero(type: Divide, x: 0)
+Microsoft.CSharp.RuntimeBinder.Tests.IntegerBinaryOperationTests.ConstantDivideByZero(type: Divide, x: 9223372036854775807)
+Microsoft.CSharp.RuntimeBinder.Tests.IntegerBinaryOperationTests.ConstantDivideByZero(type: Modulo, x: 1)
+Microsoft.CSharp.RuntimeBinder.Tests.IntegerBinaryOperationTests.ConstantDivideByZero(type: Modulo, x: -1)
+Microsoft.CSharp.RuntimeBinder.Tests.IntegerBinaryOperationTests.ConstantDivideByZero(type: Divide, x: 0)
+Microsoft.CSharp.RuntimeBinder.Tests.IntegerBinaryOperationTests.ConstantDivideByZero(type: Modulo, x: 9223372036854775807)
+Microsoft.CSharp.RuntimeBinder.Tests.IntegerBinaryOperationTests.ConstantDivideByZero(type: Divide, x: 1)
+Microsoft.CSharp.RuntimeBinder.Tests.IntegerBinaryOperationTests.ConstantDivideByZero(type: Modulo, x: 18446744073709551615)
+Microsoft.CSharp.RuntimeBinder.Tests.IntegerBinaryOperationTests.ConstantDivideByZero(type: Modulo, x: -2147483648)
+Microsoft.CSharp.RuntimeBinder.Tests.IntegerBinaryOperationTests.IntegerDivideByLiteralZero
```

Comment:
Approving, but I'm stupid and should make the ignoring of parameter base tests on xUnit better.